benchexec/pqos.py

```diff
@@ -0,0 +1,100 @@
+# BenchExec is a framework for reliable benchmarking.
+# This file is part of BenchExec.
+#
+# Copyright (C) 2007-2015  Dirk Beyer
+# All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+    This module contains the Pqos class which is used to interact with pqos_wrapper cli
+    to allocate equal cache for each thread and isolate cache of two individual threads.
+"""
+
+import os
+import logging
+import json
+from subprocess import check_output, CalledProcessError, STDOUT
+from benchexec.util import find_executable
+
+
+class Pqos(object):
+    """
+        The Pqos class defines methods to interact with pqos_wrapper cli.
+    """
+
+    CMD = "pqos_wrapper"
+    DEBUG = 0
+    WARNING = 1
```

Comment:
What is this?

Reply:
Sorry, this was used by me for some testing...i forgot to remove it in the pull request